p2p/transport/webtransport/transport_test.go

```diff
@@ -178,7 +178,7 @@ func TestHashVerification(t *testing.T) {
 		var trErr *quic.TransportError
 		require.ErrorAs(t, err, &trErr)
 		require.Equal(t, quic.TransportErrorCode(0x12a), trErr.ErrorCode)
-		require.Contains(t, trErr.ErrorMessage, "cert hash not found")
+		require.Contains(t, trErr.Error(), "cert hash not found")
```

Comment:
@marten-seemann I don't if this change was intended or not by the quic-go error change.

Reply:
Yeah, this will be resolved by #2506. Probably not the most intuitive quic-go API, maybe we need to expose an `Error error` value.

Reply:
Resolved by rebasing.